src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -206,14 +206,25 @@ private function addRoute(RouteCollection $routeCollection, string $resourceClas
         $path = trim(trim($resourceMetadata->getAttribute('route_prefix', '')), '/');
         $path .= $this->operationPathResolver->resolveOperationPath($resourceShortName, $operation, $operationType, $operationName);
 
+        $inputClass = $resourceMetadata->getAttribute('input_class', $resourceClass);
+        $outputClass = $resourceMetadata->getAttribute('output_class', $resourceClass);
+
+        if (OperationType::ITEM === $operationType) {
+            $inputClass = $resourceMetadata->getItemOperationAttribute($operationName, 'input_class', $inputClass);
```

Comment:
```suggestion
            $inputClass = $resourceMetadata->getItemOperationAttribute($operationName, 'input_class', $resourceClass, true);
```

you could get rid of the lines 209-210 and use this syntax everywhere I think (inlining the specification of the default value and the attribute fallback)

Reply:
nice, didn't notice that. thanks